app/src/features/FhirAttributePanel/StaticInput.tsx

```diff
@@ -1,16 +1,35 @@
-import React, { useState } from "react";
+import React, { useMemo, useState } from "react";
```

Comment:
I'm not sure whether this would be better. For each of the input types, there is always a textbox displayed to let the user fill manually the static value. If we made 3 different components for those 3 kinds of inputs, the textboxes would be duplicated ..?

Reply:
OK agreed, I'm fine with the current solution then